### DISCUSSION TITLE
Welcome to moshell Discussions!

### DISCUSSION THREAD

**Discussion by Override-6** [moshell-lang/moshell#54] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded.

**Comment by SH1F7ER** [comment]
Hello,

Actually there is existing a "Moshell" language used in telecommunications by Ericsson, where "mo" comes from "managed object"
https://www.scribd.com/document/649914236/AMOS-MoShell-understanding-the-basics-sessions-1-2-combined
